new_course/1_programming_fundementals/basic_enumerable_methods.md

```diff
@@ -416,6 +414,7 @@ invited_friends(friends)
 ~~~
 
 ### Assignment
+
```

Comment:
This was a mistake. The space should be between the `div` tag and the first listed item. It's corrected in the following commit.